api/serializers/substance.py

```diff
@@ -115,6 +115,9 @@ class SubstanceModificationSerializer(CommonIngredientModificationSerializer, Wi
     max_quantities = SubstanceMaxQuantityModificationSerializer(
         many=True, source="maxquantityperpopulationrelation_set", required=False
     )
+    must_specify_quantity = serializers.BooleanField(
+        source="ca_must_specify_quantity", required=False, allow_null=True
```

Comment:
pourquoi `allow_null=True` ? Je vois que le champ dans le modèle n'accepte pas de null : 
```python
    siccrf_must_specify_quantity = models.BooleanField(
        default=False, verbose_name="spécification de quantité obligatoire (selon la base SICCRF)"
    )
```

En faisant un test vite fait en mettant null j'ai : _django.db.utils.IntegrityError: null value in column "siccrf_must_specify_quantity" of relation "data_substance" violates not-null constraint_

Reply:
https://github.com/betagouv/complements-alimentaires/pull/1945/commits/e1c888380d00e12cdc6a3189c5a33a841626c952